{
  "comments": [
    {
      "unresolved": false,
      "key": {
        "uuid": "ebe6c119_e7669f83",
        "filename": "/PATCHSET_LEVEL",
        "patchSetId": 1
      },
      "lineNbr": 0,
      "author": {
        "id": 31042
      },
      "writtenOn": "2023-11-06T21:05:34Z",
      "side": 1,
      "message": "CI failure looks like a flake?",
      "revId": "159d21ece38df32aa1f6d4bce6d73c08e142914b",
      "serverId": "e5514cf8-2d6e-3e29-adb4-24cd6dde4bf0"
    }
  ]
}